pom.xml

```diff
@@ -109,7 +109,7 @@
         <curator.version>2.12.0</curator.version>
         <delta.version>1.2.1</delta.version>
         <fb303.version>0.9.3</fb303.version>
-        <flink.version>1.14.4</flink.version>
+        <flink.version>1.15.0</flink.version>
```

Comment:
We need add a `flink-1.15` maven profile to overwrite the `flink.version` property rather than upgrade it in place, and once we implement support everything on the new Flink version, we can change the default property here.

Reply:
thx for you review. this time only flink 1.15 is supported. If flink 1.15 and multiple versions are supported at the same time, the development and review time of pr will be extended. For the time being, it can be split into two sub-tasks. This time, flink 1.15 will be supported first, and we will wait for this pr After passing, you can open a file about flink multi-version support.